utils/delivery-option.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Utility for the `n-conversion-forms/partial/delivery-option.html` partial
+ * @example
+ * const deliveryOption = new DeliveryOption(document);
+ */
+class DeliveryOption {
+	/**
+	 * Initalise the Email utility
```

Comment:
Email utility? copy paste leftover?
```suggestion
	 * Initalise the DeliveryOption utility
```

Reply:
🤦‍♂ Oops... good catch